### PR TITLE
PG added basic datatypes

### PR DESCRIPTION
**Pull request by albertlast**
Signed-off-by: albertlast albertlast@hotmail.de

### CODE REVIEW

File: Sources/DbPackages-postgresql.php [hunk 1]
```diff
@@ -661,6 +661,9 @@ function smf_db_calculate_type($type_name, $type_size = null, $reverse = false)
 			'mediumtext' => 'text',
 			'largetext' => 'text',
 			'inet' => 'inet',
+			'time' => 'time without time zone',
+			'datetime' => 'timestamp without time zone',
+			'timestamp' => 'timestamp without time zone',
```
Comment (jdarwood007) [inline Sources/DbPackages-postgresql.php:666]: Is it a mistake that both timestamp and datetime have the same value?

Comment (albertlast) [reply Sources/DbPackages-postgresql.php:666]: Sure in mysql are they nearly the same too : http://dev.mysql.com/doc/refman/5.7/en/datetime.html
only the min max values is different and pg only know one datatype: https://www.postgresql.org/docs/9.6/static/datatype-datetime.html

Comment (jdarwood007) [reply Sources/DbPackages-postgresql.php:666]: Ok, just making sure.

**Comment by colinschoen** [comment]
LGTM

**Pull request merged**